test/load_data_test.js

```diff
@@ -1,83 +1,116 @@
+const dataUrls = MOCKED_DATA_URL.split(',');
+
+function dataHostname(dataUrl) {
+  return `http://${url.parse(dataUrl.split('|')[0]).hostname}`;
+}
 
-var data_hostname = "http://" + url.parse(process.env.DATA_URL).hostname;
-console.log("Host: " + data_hostname);
-var data_path = url.parse(process.env.DATA_URL).pathname;
-console.log("Path: " + data_path);
+function dataPath(dataUrl) {
+  return url.parse(dataUrl.split('|')[0]).pathname;
+}
+
+dataUrls.forEach((dataUrl) => {
+  console.log('Host: ', dataHostname(dataUrl), 'Path: ', dataPath(dataUrl));
+});
 
-describe("Loading of Data", function() {
-  beforeEach(function() {
-    var time = new Date("2016-03-01T12:00:00"); // Freeze
+describe('Loading of Data', () => {
+  beforeEach(() => {
+    const time = new Date('2016-03-01T12:00:00'); // Freeze
     tk.freeze(time);
   });
 
-  afterEach(function(){
+  afterEach(() => {
     tk.reset();
   });
 
-  describe("With a 404 on the CSV", function() {
-    nock(data_hostname)
-        .get(data_path)
+  describe('With a 404 on the CSV', () => {
+    dataUrls.forEach((dataUrl) => {
+      nock(dataHostname(dataUrl))
+        .get(dataPath(dataUrl))
         .reply(404);
+    });
 
-    it("hits the error callback with a 404 message", function() {
-      return require("../utils/loaddata")().then(assert.failed, function(err) {
-        expect(err).to.include("404 page not found");
+    it('hits the error callback with a 404 message', () => {
+      return loadData(MOCKED_DATA_URL).then(assert.failed, (err) => {
+        expect(err.message).to.include('404 page not found');
       });
     });
   });
 
-  describe("With a 200 on the CSV", function() {
-    beforeEach(function() {
-      nock(data_hostname)
-          .get(data_path)
-          .reply(200, function() {
-            return fs.createReadStream('test/fixtures/acs_mo_event.csv');
-          });
+  describe('With a 200 on the CSV', () => {
+    beforeEach(() => {
+      dataUrls.forEach((dataUrl) => {
+        const path = dataPath(dataUrl);
+        nock(dataHostname(dataUrl))
+          .get(path)
+          .reply(200, () => fs.createReadStream(`test/fixtures/${path.split('/').reverse()[0]}`));
+      });
     });
 
-    it("hits the success callback correctly", function() {
-      return require("../utils/loaddata")()
-      .then(resp => { expect(resp).to.equal(true)})
+    it('hits the success callback correctly', () => {
+      return loadData(MOCKED_DATA_URL)
+        .then((resp) => { expect(resp).to.equal(true); });
     });
 
-    it("creates 36 cases", function() { // 38 lines, two sets of duplicates
-      return require("../utils/loaddata")()
-        .then(resp => knex("cases").count('* as count'))
-        .then(rows => {
-          expect(rows[0].count).to.equal('36');
-        })
+    it('creates 55 cases', () => {
```

Comment:
I love the arrow functions, but I wonder if we should follow Mocha's advice, which discourages them in it's test functions because Mocha binds `this` to the test context. It doesn't seem like any of our tests use Mocha's context or `this`, so it's probably a non-issue, but might be in the future. I don't feel strongly one way or the other.

Reply:
@markmeyerphoto I can revert those if you want... I'll need to read up on Mocha...

Reply:
Your call @mfrederickson. I really don't think we'll ever be passing context around this way — I always just declare variables at the top of describe blocks, but I think some people like to assign things to `this`. Here's the relevant mocha bit: https://mochajs.org/#arrow-functions

Reply:
@markmeyerphoto interesting... since the example shows a timeout and I actually saw one in one of our test scripts, but not this one.  I should change them back.  I'll get to that soon...